packages/circuit-ui/components/Popover/Popover.spec.tsx

```diff
@@ -73,37 +78,43 @@ describe('Popover', () => {
     ])(
       'should open the popover when pressing the %s key on the trigger element',
       async (_, key) => {
-        renderPopover({ ...baseProps, initialOpen: false });
+        renderPopover({ ...baseProps, isOpen: false });
 
         const popoverTrigger = screen.getByRole('button');
 
         popoverTrigger.focus();
         await userEvent.keyboard(key);
 
-        expect(screen.getByText(popoverContent)).toBeVisible();
+        expect(baseProps.onToggle).toHaveBeenCalledTimes(1);
       },
     );
   });
 
   describe('when open', () => {
+    it('should render its content', () => {
+      renderPopover(baseProps);
+
+      expect(screen.getByText(popoverContent)).toBeVisible();
+    });
+
     it('should close the popover when clicking outside', async () => {
       renderPopover(baseProps);
 
       await userEvent.click(document.body);
 
       await waitFor(() => {
-        expect(baseProps.onClose).toHaveBeenCalledTimes(1);
+        expect(baseProps.onToggle).toHaveBeenCalledTimes(1);
       });
     });
 
     it('should close the popover when clicking the trigger element', async () => {
       renderPopover(baseProps);
 
-      const popoverTrigger = screen.getByRole('button');
+      const popoverTrigger = screen.getByText('Button');
```

Comment:
Semantic queries better reflect how a user would interact with an element. Use the `name` option to narrow down which button to get. 

```suggestion
      const popoverTrigger = screen.getByRole('button', { name: 'Button' });
```

(same below)